rmgpy/molecule/group.py

```diff
@@ -472,31 +471,42 @@ def isSurfaceSite(self):
 
     def isOxygen(self):
         """
-        Return ``True`` if the atom represents an oxygen atom or ``False`` if
-        not.
+        Return ``True`` if the atom represents an oxygen atom or ``False`` if not.
         """
         allOxygens = [atomTypes['O']] + atomTypes['O'].specific
-        checkList=[x in allOxygens for x in self.atomType]
-
+        checkList = [x in allOxygens for x in self.atomType]
         return all(checkList)
 
     def isSulfur(self):
         """
-        Return ``True`` if the atom represents an sulfur atom or ``False`` if
-        not.
+        Return ``True`` if the atom represents an sulfur atom or ``False`` if not.
         """
         allSulfur = [atomTypes['S']] + atomTypes['S'].specific
-        checkList=[x in allSulfur for x in self.atomType]
+        checkList = [x in allSulfur for x in self.atomType]
+        return all(checkList)
 
+    def isNitrogen(self):
+        """
+        Return ``True`` if the atom represents an sulfur atom or ``False`` if not.
+        """
+        allNitrogen = [atomTypes['N']] + atomTypes['N'].specific
+        checkList = [x in allNitrogen for x in self.atomType]
+        return all(checkList)
+
+    def isCarbon(self):
+        """
+        Return ``True`` if the atom represents an sulfur atom or ``False`` if not.
+        """
+        allCarbon = [atomTypes['C']] + atomTypes['C'].specific
+        checkList = [x in allCarbon for x in self.atomType]
         return all(checkList)
 
     def hasWildcards(self):
         """
         Return ``True`` if the atom has wildcards in any of the attributes:
-        atomtype, electronpairs, lone pairs, charge, and bond order. Returns
+        atomtype, lone pairs, charge, and bond order. Returns
```

Comment:
I think it was probably meant to say radical electrons.

Reply:
I agree, changed.
Also, the docstring says "charge", but the method doesn't check for it

Reply:
It seems you squashed the fix into the other commit. So the first commit removes electronpairs and the second commit adds radical electrons. It would be best if you could move it to one commit.

Reply:
oops! fixed